src/App.jsx

```diff
@@ -14,70 +15,79 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const activeGoodClass = 'has-background-success-light';
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const onAddHandler = good => setSelectedGood(good);
+  const onClearHandler = () => setSelectedGood('');
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {
+          !selectedGood.length
+            ? (
+              'No goods selected'
+            )
+            : (
+              <>
+                {`${selectedGood} is selected`}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+                <button
+                  data-cy="ClearButton"
+                  type="button"
+                  className="delete ml-3"
+                  onClick={onClearHandler}
+                />
+              </>
+            )
+        }
+      </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+      <table className="table">
+        <tbody>
+          {
+            goods.map(good => (
+              <tr
+                data-cy="Good"
+                key={good}
+                className={good === selectedGood ? activeGoodClass : ''}
+              >
+                <td>
+                  {
+                    good === selectedGood
+                      ? (
+                        <button
+                          data-cy="RemoveButton"
+                          type="button"
+                          className="button is-info"
+                          onClick={onClearHandler}
+                        >
+                          -
+                        </button>
+                      )
+                      : (
+                        <button
+                          data-cy="AddButton"
+                          type="button"
+                          className="button"
+                          onClick={() => onAddHandler(good)}
+                        >
+                          +
+                        </button>
```

Comment:
I'd prefer using one button with different attributes, text and logic for onclick